ci/ci_test.bat

```diff
@@ -0,0 +1,44 @@
+@echo off
+
+setlocal
+
+REM Keep in mind to run ci/ci_setup.sh if you need to setup/clean up your environment before
```

Comment:
@jsvd I guess this would be a different script for Windows? I guess we could just delete the comment for now?